dune

```diff
@@ -3,7 +3,7 @@
     (ocamlopt_flags (:standard -g -p -w -39))
     (flags (:standard -w -39))
   )
-  (dev (flags (:standard -g -w -39)))
+  (dev (flags (:standard -g -w -39 -warn-error -32-38-67-69)))
```

Comment:
My prediction is we'll forget about these and will be left disabled

Reply:
You do see the warnings, they are just not errors (which is why I didn't completely turn off the warnings). When I have some time I'll try to clean them up, but there were too many for a friday.